Files/Interacts/BaseLayoutCommandImplementationModel.cs

```diff
@@ -420,7 +421,10 @@ public virtual async void ItemPointerPressed(PointerRoutedEventArgs e)
                     }
                     else
                     {
-                        await NavigationHelpers.OpenPathInNewTab(Item.ItemPath);
```

Comment:
What does this change?

Reply:
Previously when you press middle button whatever the layout you are using that code open new that:
`await NavigationHelpers.OpenPathInNewTab(Item.ItemPath);`

So, i added an limitation to just open folders on a new tab if that layout be ColumnView (`ColumnViewBrowser`):
`if (SlimContentPage is ColumnViewBrowser) { await NavigationHelpers.OpenPathInNewTab(Item.ItemPath); }`

Reply:
It's meant to open in other layouts too